File: Arrays/Python/String_Manipulation.py
class Solution:
    def removeAdj(self,v,n):
        # Your code goes here
        stack = []
        for item in v:
            if len(stack)==0:
                stack.append(item)
                continue
            else:
                if item == stack[-1]:
                    stack.pop()
                else:
                    stack.append(item)
        return len(stack)

File: Solutions/Java/String_Manipulation.java
class Solution {
    static int removeConsecutiveSame(Vector <String > v) {
        // Your code goes here
        Stack<String> st = new Stack<>();
        for (String x : v) {
            if (st.isEmpty()) {
                st.push(x);
            }
            else {
                if (st.peek().equals(x)) {
                    st.pop();
                }
                else {
                    st.push(x);
                }
            }
        }
        return st.size();
    }
}
